viewing_party/party.py

```diff
@@ -0,0 +1,154 @@
+from collections import Counter
+
+
+def create_movie(title, genre, rating):
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
```

Comment:
The project description indicates that we should return None if any of these inputs are _falsy_. `None` is  a falsy value, but it's not the only one. To handle any falsy value, consider writing this as
```python
    if not title or not genre or not rating:
        return None
```